### DISCUSSION TITLE
Account restricted

### DISCUSSION THREAD

**Discussion by shiftpsh** [codewars/codewars.com#2975] [Q&A] [answered]
Hello. I am Suhyun Park, former ICPC world finalist representing Sogang University — Republic of Korea.

My account got restricted for violation of TOS and I think it is by an error. Is there any way that I can appeal?

My handle is `shiftpsh` on codewars, and I can verify my problem solving competition history etc. if needed.

**Comment by Blind4Basics** [comment, marked as answer]
Handled. Sorry for the troubles...

**Reply by Blind4Basics** [reply]
side note: what are the emojis for, in your solutions?

**Reply by shiftpsh** [reply]
Thanks!

The emojis are part of my template - symbolizing the [ICPC logo](https://www.google.com/url?sa=i&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FInternational_Collegiate_Programming_Contest&psig=AOvVaw19mMmXNJmTLTCIeKYll7Qo&ust=1680022968890000&source=images&cd=vfe&ved=0CBAQjRxqFwoTCIiIjLTL_P0CFQAAAAAdAAAAABAE). I started to use this template since I was practicing for ICPC, and felt it was quite fancy, so it became my routine.
